.github/workflows/aws-deploy.yaml

```diff
@@ -0,0 +1,76 @@
+name: aws-deploy
+
+on:
+  pull_request:
+    branches: '*'
+  push:
+    branches: [main, develop]
+
+jobs:
+
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: pre-commit/action@v2.0.0
+
+  sceptre-deploy:
+    if: github.event_name == 'push'
+    runs-on: ubuntu-latest
+    needs: pre-commit
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Install pipenv
```

Comment:
maybe you want to use the github actions instead? https://github.com/marketplace/actions/pipenv-for-github-actions

Reply:
I'll give it a try! We'll have to confirm that it interacts well with the `aws-actions/configure-aws-credentials@v1` action.